setup/ubuntu14.04/ansible-playbook.yaml

```diff
@@ -1,7 +1,6 @@
 ---
 - hosts: iojs-build-ubuntu1404
-
-  remote_user: root
+  sudo: yes
```

Comment:
@jbergstroem remote user on these machines isn't root, I'd like it to log in and `sudo` but I know other servers we're using log in as `root`, any ideas on how we can reconcile the two?